content/desktop/networking.md

```diff
@@ -101,10 +108,10 @@ ping the Windows containers.
 
 ### Per-container IP addressing is not possible
 
-The docker bridge network is not reachable from the host.
+The Docker-bridge network is not reachable from the host.
 However if you are a Windows user, it works with Windows containers.
```

Comment:
suggestion: These two sentences read odd together like this.

- The first is about the network not being accessible from the host
- The second says it "works" (what is it that works? is it reachable from the host if you use hyper-v?)

```suggestion
The Docker `bridge` network is not reachable from the host.
However if you are a Windows user, it works with Windows containers.
```